scipy-1.0/paper.tex

```diff
@@ -401,55 +362,9 @@ \subsection*{SciPy Begins}
 scale.  Unfortunately, NumArray proved inefficient for small arrays,
 presenting the community with a rather unfortunate choice.  In 2005,
 Travis Oliphant combined the best elements of Numeric and NumArray,
-thereby solving the dilemma and paving the way for Python to become
-a very significant player in the Data Science movement. NumPy
-1.0 was released in October 2006\cite{numpy-1.0-tag}, with about 30
-authors recognized for major contributions in its release notes.
-
-% Future directions for SciPy after meeting at Berkeley
-% https://mail.python.org/pipermail/scipy-user/2005-March/004164.html
-
-%In April 2006, Paul
-%Dubois\footnote{https://mail.python.org/pipermail/numpy-discussion/2006-April/007487.html} wrote
-%to the NumPy mailing list:
-%\begin{quote}
-%IEEE's magazine, Computing in Science and Engineering (CiSE), has asked me
-%to put together a theme issue on the use of Python in Science and
-%Engineering. I will write an overview to be accompanied by 3-5 articles [. . .]
-%that show a diverse set of applications or
-%tools, to give our readers a sense of whether or not Pyth%on might be useful
-%in their own work.
-%\end{quote}
-In a May/June 2007 special issue of IEEE Computing in Science and
```

Comment:
The special issue is mentioned in "SciPy Matures". If there are really important parts of this that we want to keep, we can add them back there.